fuzzamoto-ir/src/generators/block.rs

```diff
@@ -76,7 +76,10 @@ impl<R: RngCore> Generator<R> for BlockGenerator {
             vec![conn_var.index, block_and_header_var[1].index],
             Operation::SendBlock,
         );
-        builder.force_append(vec![block_and_header_var[2].index], Operation::TakeTxo);
+        builder.force_append(
+            vec![block_and_header_var[2].index],
+            Operation::TakeCoinbaseTxo,
```

Comment:
Just leaving a note, this does not need to be done in this PR. The coinbase isn't mature yet so sometimes transactions will be invalid (https://github.com/bitcoin/bitcoin/blob/d5c8199b7904f4b13230d5677d4528591e4d5b41/src/consensus/tx_verify.cpp#L180). I think fixing this requires info about the node's block height and when it's ok to call `TakeCoinbaseTxo`?